service/api.go

```diff
@@ -322,7 +361,8 @@ func (as *APIServer) GetServiceApp() *gimlet.APIApp {
 	app.SimpleVersions = true
 
 	// Project lookup and validation routes
-	app.AddRoute("/ref/{identifier}").Wrap(requireUser).Handler(as.fetchProjectRef).Get()
+	app.AddRoute("/ref/{projectId}").Wrap(requireUser).Handler(as.fetchLimitedProjectRef).Get()
+	app.AddRoute("/ref/{projectId}/workstation").Wrap(requireUser).Handler(as.projectWithWorkstationConfig).Get()
```

Comment:
Rather than add more routes to the long-deprecated legacy UI, I would just turn the workstation route into a REST v2 route in `rest/route/service.go` (and possibly the limited project ref route as well).

Reply:
done, added it up top